source/lex.tex

```diff
@@ -362,7 +362,7 @@
 
 \item Otherwise,
 the next preprocessing token is the longest sequence of
-characters that could constitute a preprocessing token, even if that
+characters that would constitute a preprocessing token, even if that
```

Comment:
I'm not sure "would" works here. Second options? @zygoloid?

Reply:
We have a bunch of max-munch rules that all follow the same pattern.

Reply:
OK, then maybe this is fine as is (likewise below).

Reply:
What I've been suggesting in other PRs is:
"longest sequence of characters that matches the syntax of a *preprocessing-token*"
... which seems more precise than our prior wording in the various tentative parsing areas, and distinguishes between matching the syntax of X and actually forming a valid X, without getting into hypotheticals.

Reply:
fixed